### PR TITLE
Convert to RISC-V template (#68)

### PR DESCRIPTION
**Pull request by jjscheel**
Signed-off-by: Jeff Scheel <jeff@riscv.org>

### CODE REVIEW

**Comment by jjscheel** [comment]
@timsifive, after many iterations of the [actions.yml](https://github.com/riscv/configuration-structure/blob/master/.github/workflows/actions.yml) file based on the [docs-spec-template](https://github.com/riscv/docs-spec-template) similar file, [build-pdf.yml](https://github.com/riscv/docs-spec-template/blob/main/.github/workflows/build-pdf.yml), I finally got things working.

Take a look and let me know if you have any questions.

File: Makefile [hunk 1]
```diff
@@ -10,7 +10,13 @@ draft:	$(DRAFT).pdf
...
 
 %.pdf: %.adoc
-	asciidoctor-pdf $<
+	asciidoctor-pdf \
+	-a toc \
+	-a compress \
+	-a pdf-style=docs-resources/themes/riscv-pdf.yml \
+	-a pdf-fontsdir=docs-resources/fonts \
+	-o $@ $<
+#	asciidoctor-pdf $<
```
Comment (timsifive) [inline Makefile]: Can you remove this commented out line?

File: riscv-configuration-structure-draft.adoc [hunk 2]
```diff
@@ -1,19 +1,64 @@
...
+:source-highlighter: coderay
+endif::[]
+:data-uri:
+:hide-uri-scheme:
+:stem: latexmath
+:footnote:
+:xrefstyle: short
 
-= RISC-V Configuration Description Version {version}
+= RISC-V Configuration Description
```
Comment (timsifive) [inline riscv-configuration-structure-draft.adoc:34]: Looks like version was renamed to revnumber. That's fine, but shouldn't it be in the document somewhere? Or is there some magic that inserts it somewhere?

**Review by timsifive — CHANGES_REQUESTED** [review]
Wow. Looks like you made it happy without even changing the actions. Thank you.

Looks good, but I do have a few minor nits below.

File: Makefile (hunk 1 above)
Comment (jjscheel) [reply Makefile]: Yes, I'll close this PR and resubmit.

File: riscv-configuration-structure-draft.adoc (hunk 2 above)
Comment (jjscheel) [reply riscv-configuration-structure-draft.adoc:34]: Revnumber is included in the PDF automatically.  If you add it to the title as you've done, your title page will look REALLY odd.  However, if you remove it as I've done, you cannot tell the revnumber from the Web viewed file.

Thoughts?

Comment (jjscheel) [reply riscv-configuration-structure-draft.adoc:34]: I will put it back in and submit the PR.  Then, you can do any cleanup you'd like.

**Comment by jjscheel** [comment]
Closing to submit a clean PR with single commit.

**Comment by timsifive** [comment]
> Closing to submit a clean PR with single commit.

I appreciate your commitment to commit history, but I would have done a squash merge in any case so this is unnecessary. It's also not necessary to make a new PR even if you do want to rewrite the history. Instead you can rewrite the history locally, and force push.

File: riscv-configuration-structure-draft.adoc (hunk 2 above)
Comment (timsifive) [reply riscv-configuration-structure-draft.adoc:34]: If it's included automatically that's fine, and you can disregard my comment. Although I do worry that some people would get hold of the adoc and not the pdf.

**Pull request closed without merging**